### DISCUSSION TITLE
Next Cloud NAS instal

### DISCUSSION THREAD

**Discussion by Motocycle347** [szaimen/Nextcloud-NAS-Guide#74] [General] [answered]
Followed all instructions On page 28 after the initial script the server did not reboot. and I can not continue.I can try again or is there a way to start the second part ot the script? I am running on VMware® Workstation 16 Pro

**Comment by szaimen** [comment]
Hi, what was the issue why it failed? Can you post a screenshot? or a logfile or so?

**Reply by Motocycle347** [reply]
stopped so i rebooted what log file can i send?

**Reply by szaimen** [reply]
but why did it stop there? did you see an issue message or something like that anywhere?

**Reply by Motocycle347** [reply]
no message? just normal line prompt

**Reply by szaimen** [reply]
so nothing we could use to find out why it stopped?

**Reply by szaimen** [reply]
Either way you should probably be able to continue with the whole setup by running

```
sudo rm "/var/scripts/you-can-not-run-the-startup-script-several-times"
sudo rm "/var/scripts/nextcloud-startup-script.sh"
```

**Reply by Motocycle347** [reply]
Not that I know but I can try again

**Comment by Motocycle347** [comment]
Didnt have that script to remove . I am I'm going to start again from scratch

**Comment by Motocycle347** [comment]
error messages 2nd try

**Reply by szaimen** [reply]
> error messages 2nd try

What were the error messages? I cannot see them...

**Comment by Motocycle347** [comment]
[⣾⣽⣻⣾⣽⣻⣾⣽⣻] 
Reading package lists... Done 
Building dependency tree 
Reading state information... Done 
E: Unable to locate package php8.1-fpm 
E: Couldn't find any package by glob 'php8.1-fpm' 
E: Couldn't find any package by regex 'php8.1-fpm' 
E: Unable to locate package php8.1-intl 
E: Couldn't find any package by glob 'php8.1-intl' 
E: Couldn't find any package by regex 'php8.1-intl' 
E: Unable to locate package php8.1-ldap 
E: Couldn't find any package by glob 'php8.1-ldap' 
E: Couldn't find any package by regex 'php8.1-ldap' 
E: Unable to locate package php8.1-imap 
E: Couldn't find any package by glob 'php8.1-imap' 
E: Couldn't find any package by regex 'php8.1-imap' 
E: Unable to locate package php8.1-gd 
E: Couldn't find any package by glob 'php8.1-gd' 
E: Couldn't find any package by regex 'php8.1-gd' 
E: Unable to locate package php8.1-pgsql 
E: Couldn't find any package by glob 'php8.1-pgsql' 
E: Couldn't find any package by regex 'php8.1-pgsql' 
E: Unable to locate package php8.1-curl 
E: Couldn't find any package by glob 'php8.1-curl' 
E: Couldn't find any package by regex 'php8.1-curl' 
E: Unable to locate package php8.1-xml 
E: Couldn't find any package by glob 'php8.1-xml' 
E: Couldn't find any package by regex 'php8.1-xml' 
E: Unable to locate package php8.1-zip 
E: Couldn't find any package by glob 'php8.1-zip' 
E: Couldn't find any package by regex 'php8.1-zip' 
E: Unable to locate package php8.1-mbstring 
E: Couldn't find any package by glob 'php8.1-mbstring' 
E: Couldn't find any package by regex 'php8.1-mbstring' 
E: Unable to locate package php8.1-soap 
E: Couldn't find any package by glob 'php8.1-soap' 
E: Couldn't find any package by regex 'php8.1-soap' 
E: Unable to locate package php8.1-gmp 
E: Couldn't find any package by glob 'php8.1-gmp' 
E: Couldn't find any package by regex 'php8.1-gmp' 
E: Unable to locate package php8.1-bz2 
E: Couldn't find any package by glob 'php8.1-bz2' 
E: Couldn't find any package by regex 'php8.1-bz2' 
E: Unable to locate package php8.1-bcmath 
E: Couldn't find any package by glob 'php8.1-bcmath' 
E: Couldn't find any package by regex 'php8.1-bcmath' 
Sorry but something went wrong. Please report this issue to https://github.com/nextcloud/vm/issues and include the output of the error message. Thank you! 
apt-get install -y php8.1-fpm php8.1-intl php8.1-ldap php8.1-imap php8.1-gd php8.1-pgsql php8.1-curl php8.1-xml php8.1-zip php8.1-mbstring php8.1-soap php8.1-gmp php8.1-bz2 php8.1-bcmath php-pear failed 
sudo: php: command not found 
Checking connection... 
Home/SME Server not detected. No worries, just testing the function. 
Extending LVM, this may take a long time... 
New size (11007 extents) matches existing size (11007 extents). 
<43.00 
5.00 
30.00 
resize2fs 1.45.5 (07-Jan-2020) 
The filesystem is already 11271168 (4k) blocks long. Nothing to do! 

postgresql is installed, it must be a clean server. 
***@***.***:~$ E: Unable to locate package php8.1-fpm 
E: Unable to locate package php8.1-xml 
E: Couldn't find any package by glob 'php8.1-xml' 
E: Couldn't find any package by regex 'php8.1-xml' 
E: Unable to locate package php8.1-zip 
E: Couldn't find any package by glob 'php8.1-zip' 
E: Couldn't find any package by regex 'php8.1-zip' 
E: Unable to locate package php8.1-mbstring 
E: Couldn't find any package by glob 'php8.1-mbstring' 
E: Couldn't find any package by regex 'php8.1-mbstring' 
E: Unable to locate package php8.1-soap 
E: Couldn't find any package by glob 'php8.1-soap' 
E: Couldn't find any package by regex 'php8.1-soap' 
E: Unable to locate package php8.1-gmp 
E: Couldn't find any package by glob 'php8.1-gmp' 
E: Couldn't find any package by regex 'php8.1-gmp' 
E: Unable to locate package php8.1-bz2 
E: Couldn't find any package by glob 'php8.1-bz2' 
E: Couldn't find any package by regex 'php8.1-bz2' 
E: Unable to locate package php8.1-bcmath 
E: Couldn't find any package by glob 'php8.1-bcmath' 
E: Couldn't find any package by regex 'php8.1-bcmath' 
Sorry but something went wrong. Please report this issue to https://github.com/nextcloud/vm/issues and include the output of the error message. Thank you! 
apt-get install -y php8.1-fpm php8.1-intl php8.1-ldap php8.1-imap php8.1-gd php8.1-pgsql php8.1-curl php8.1-xml php8.1-zip php8.1-mbstring php8.1-soap php8.1-gmp php8.1-bz2 php8.1-bcmath php-pear failed 
sudo: php: command not found 
Checking connection... 
Home/SME Server not detected. No worries, just testing the function. 
Extending LVM, this may take a long time... 
New size (11007 extents) matches existing size (11007 extents). 
<43.00 
5.00 
30.00 
resize2fs 1.45.5 (07-Jan-2020) 
The filesystem is already 11271168 (4k) blocks long. Nothing to do! 

postgresql is installed, it must be a clean server. 
***@***.***:~$ 
E:: command not found 
***@***.***:~$ E: Couldn't find any package by glob 'php8.1-fpm' 
> E: Couldn't find any package by regex 'php8.1-fpm' 
E:: command not found 
***@***.***:~$ E: Unable to locate package php8.1-intl 
E:: command not found 
***@***.***:~$ E: Couldn't find any package by glob 'php8.1-intl' 
> E: Couldn't find any package by regex 'php8.1-intl' 
E:: command not found 
***@***.***:~$ E: Unable to locate package php8.1-ldap 
E:: command not found 
***@***.***:~$ E: Couldn't find any package by glob 'php8.1-ldap' 
> E: Couldn't find any package by regex 'php8.1-ldap' 
E:: command not found 
***@***.***:~$ E: Unable to locate package php8.1-imap 
E:: command not found 
***@***.***:~$ E: Couldn't find any package by glob 'php8.1-imap' 
> E: Couldn't find any package by regex 'php8.1-imap' 
E:: command not found 
***@***.***:~$ E: Unable to locate package php8.1-gd 
E:: command not found 
***@***.***:~$ E: Couldn't find any package by glob 'php8.1-gd' 
> E: Couldn't find any package by regex 'php8.1-gd' 
E:: command not found 
***@***.***:~$ E: Unable to locate package php8.1-pgsql 
E:: command not found 
***@***.***:~$ E: Couldn't find any package by glob 'php8.1-pgsql' 
> E: Couldn't find any package by regex 'php8.1-pgsql' 
E:: command not found 
***@***.***:~$ E: Unable to locate package php8.1-curl 
E:: command not found 
***@***.***:~$ E: Couldn't find any package by glob 'php8.1-curl' 
> E: Couldn't find any package by regex 'php8.1-curl' 
E:: command not found 
***@***.***:~$ E: Unable to locate package php8.1-xml 
E:: command not found 
***@***.***:~$ E: Couldn't find any package by glob 'php8.1-xml' 
> E: Couldn't find any package by regex 'php8.1-xml' 
E:: command not found 
***@***.***:~$ E: Unable to locate package php8.1-zip 
E:: command not found 
***@***.***:~$ E: Couldn't find any package by glob 'php8.1-zip' 
> E: Couldn't find any package by regex 'php8.1-zip' 
E:: command not found 
***@***.***:~$ E: Unable to locate package php8.1-mbstring 
E:: command not found 
***@***.***:~$ E: Couldn't find any package by glob 'php8.1-mbstring' 
> E: Couldn't find any package by regex 'php8.1-mbstring' 
E:: command not found 
***@***.***:~$ E: Unable to locate package php8.1-soap 
E:: command not found 
***@***.***:~$ E: Couldn't find any package by glob 'php8.1-soap' 
> E: Couldn't find any package by regex 'php8.1-soap' 
E:: command not found 
***@***.***:~$ E: Unable to locate package php8.1-gmp 
E:: command not found 
***@***.***:~$ E: Couldn't find any package by glob 'php8.1-gmp' 
> E: Couldn't find any package by regex 'php8.1-gmp' 
E:: command not found 
***@***.***:~$ E: Unable to locate package php8.1-bz2 
E:: command not found 
***@***.***:~$ E: Couldn't find any package by glob 'php8.1-bz2' 
> E: Couldn't find any package by regex 'php8.1-bz2' 
E:: command not found 
***@***.***:~$ E: Unable to locate package php8.1-bcmath 
E:: command not found 
***@***.***:~$ E: Couldn't find any package by glob 'php8.1-bcmath' 
> E: Couldn't find any package by regex 'php8.1-bcmath' 
E:: command not found 
***@***.***:~$ Sorry but something went wrong. Please report this issue to https://github.com/nextcloud/vm/issues and include the output of the error message. Thank you! 
Sorry: command not found 
***@***.***:~$ apt-get install -y php8.1-fpm php8.1-intl php8.1-ldap php8.1-imap php8.1-gd php8.1-pgsql php8.1-curl php8.1-xml php8.1-zip php8.1-mbstring php8.1-soap php8.1-gmp php8.1-bz2 php8.1-bcmath php-pear failed 
E: Could not open lock file /var/lib/dpkg/lock-frontend - open (13: Permission denied) 
E: Unable to acquire the dpkg frontend lock (/var/lib/dpkg/lock-frontend), are you root? 
***@***.***:~$ sudo: php: command not found 

Command 'sudo:' not found, did you mean: 

command 'sudo' from deb sudo (1.8.31-1ubuntu1.2) 
command 'sudo' from deb sudo-ldap (1.8.31-1ubuntu1.2) 

Try: sudo apt install <deb name> 

***@***.***:~$ Checking connection... 
Checking: command not found 
***@***.***:~$ Home/SME Server not detected. No worries, just testing the function. 
-bash: Home/SME: No such file or directory 
***@***.***:~$ Extending LVM, this may take a long time... 
Extending: command not found 
***@***.***:~$ New size (11007 extents) matches existing size (11007 extents). 
-bash: syntax error near unexpected token `(' 
***@***.***:~$ <43.00 
-bash: 43.00: No such file or directory 
***@***.***:~$ 5.00 
5.00: command not found 
***@***.***:~$ 30.00 
30.00: command not found 
***@***.***:~$ resize2fs 1.45.5 (07-Jan-2020) 
-bash: syntax error near unexpected token `(' 
***@***.***:~$ The filesystem is already 11271168 (4k) blocks long. Nothing to do! 
-bash: syntax error near unexpected token `(' 
***@***.***:~$ 
***@***.***:~$ postgresql is installed, it must be a clean server. 
postgresql: command not found 
***@***.***:~$ ***@***.***:~$ 
***@***.***:~$: command not found 
***@***.***:~$

**Comment by szaimen** [comment, marked as answer]
It looks like you had bad luck and you started the installation before the script was moved to be compatible with Ubuntu 22.04 which dropped support for 20.04.

I fear you need to start here again: https://szaimen.github.io/Nextcloud-NAS-Guide/docs/ubuntu
(including downloading the new ubuntu iso file...)
